### DISCUSSION TITLE
演算法鬼斧神工，AI 畫作的神來之筆

### DISCUSSION THREAD

**Discussion by Jay-722** [1101-GS4719-C/Python-Community-GS4719-C#607] [General]
AI畫作的創造力和想像力讓人驚訝，充滿視覺衝擊力很難想像是AI幾小時就完成，更重要的是，AI繪畫工具Disco Diffusion甚至不需使用者有任何AI知識，僅需輸入幾段描述文字，一幅幅精美的畫作就會以演算法生成。

Disco Diffusion是藝術家[Somnai](https://mobile.twitter.com/somnai_dreams)設計，程式背後是CLIP人工智慧圖像生成技術，在Google Colab（雲端筆電環境）運行，使用者不需有性能強勁的設備，只要網路穩定就能使用AI繪畫工具

**全文 : https://technews.tw/2022/05/15/ai-disco-diffusion/**

**Comment by HsuChen88** [comment]
難道連畫家都要被AI取代了嗎..
用AI產生的圖片不知道有沒有版權問題
不然感覺蠻適合拿來當遊戲圖片的

**Comment by yucheng1028** [comment]
期待它的發展

**Comment by gnsJhenJie** [comment]
https://github.com/titu1994/Neural-Style-Transfer
這個專案也是做圖片風格轉換的，我自己有玩過，還蠻有趣的，專案內也有Colab的連結可以直接用Google Colab執行

**Comment by minniewei** [comment]
我覺得畫家還是不會被取代吧，畢竟情感這種東東還是難以用ai模擬出來

**Comment by 11timyyy** [comment]
雖說機器人不會像畫家一樣投入情感，但基本的勾勒其實做得還不錯!

**Comment by henrywen547** [comment]
因為AI並不會有一個自己作畫的風格，所以對於創新這個元素應該還蠻值得期待的

**Comment by awkward-willy** [comment]
這些畫作真是令人大開眼界
不過也能讓人們更加深刻地去思考到底甚麼東西能被稱之為藝術
若未來能結合時事、給定的情感等內容感覺會更加驚人

**Comment by leon10456** [comment]
感謝樓主的分享 好酷!!

**Comment by liangyinglly** [comment]
這也是圖像辨識的一環

**Comment by Lynn12378** [comment]
感覺有一天畫畫也能向機器學習了

**Comment by Jimmy911218** [comment]
我認為ai確實化的完美卻不一定能有人工才有的美才有的美

**Comment by Hchejxnc** [comment]
感謝分享~之後如果需要插畫之類的或許可以利用AI
